packages/react-dogfood/components/Stage.tsx

```diff
@@ -24,7 +25,9 @@ export const Stage = ({
   return (
     // @ts-expect-error
     <SelectedComponent
-      groupSize={!groupSize || groupSize > 16 ? undefined : groupSize}
+      {...props}
+      // @ts-expect-error
+      groupSize={!groupSize || groupSize > 16 ? props?.groupSize : groupSize}
```

Comment:
Correct, it was/is an option so I decided to keep it for now - mainly for testing purposes.

Reply:
I think it would be useful in Pronto to have the option to go lower than 16. I think we shouldn't put any constraints and use the default value (16) unless specified otherwise.

Reply:
That's what it does though... you can pass `?group_size=4` to the search params in your URL and you'll have a layout that renders only 4 participants per page - am I missing something? 😄

Reply:
If you specify number higher than 16 or 0 then it defaults to 16.